apps/hash-ai-worker-py/app/workflows.py

```diff
@@ -199,3 +201,46 @@ async def get_entity_type(
         print_schema(entity_type_model, params.entity_type)
 
         return entity_type_model.model_json_schema()
+
+
+class ProposedEntity(BaseModel, extra=Extra.forbid):
+    """An entity proposed by AI."""
+
+    entity_type_id: str = Field(..., alias="entityTypeId")
+    properties: Any
+
+
+class InferEntitiesWorkflowParameter(BaseModel, extra=Extra.forbid):
+    """Parameters for entity inference workflow."""
+
+    text_input: str = Field(..., alias="textInput")
+    entity_type_ids: list[str] = Field(..., alias="entityTypeIds")
+    actor_id: UUID = Field(..., alias="actorId")
+
+
+class InferEntitiesWorkflowResult(BaseModel, extra=Extra.forbid):
+    """Result of entity inference workflow."""
+
+    entities: list[ProposedEntity]
+
+
+@workflow.defn(name="inferEntities")
+class InferEntitiesWorkflow:
+    """A workflow that reads an entity type from the provided URL."""
```

Comment:
I'm not sure this is quite right... maybe something like this?
```suggestion
    """A workflow that infer entities of the specified type(s) from the provided text input"""
```

Reply:
Python has this incredibly stupid guideline that the first sentence must be less than 88 characters and end with a period.
What about this?
```suggestion
    """Infers entities of the specified type(s) from the provided text input."""
```